### PR TITLE
hardening/807_add_transient_option_to_GET_method

### PR DESCRIPTION
**Pull request by pcoello25**
* Partially fix issue #807 
  * Update CHANGES_NEXT_RELEASE is not neccesary
* 100% unit tests passed:
```
Results : Tests run: 66, Failures: 0, Errors: 0, Skipped: 0
```
* e2e (unofficial) tests passed:
```
[DEFAULT CASE: NO PARAMETERS -> RUN LIKE `verbose=false` and `transient=true`]
$ curl -X GET "http://localhost:8081/admin/log"
{"level": "DEBUG"}

[CASES WITH TRANSIENT AND NO VERBOSE]
$ curl -X GET "http://localhost:8081/admin/log?transient=false"
{"level": "INFO"}
$ curl -X GET "http://localhost:8081/admin/log?transient=true"
{"level": "DEBUG"}

[CASES WITH VERBOSE AND NO TRANSIENT]
$ curl -X GET "http://localhost:8081/admin/log?verbose=true"
{"success":"true","log4j":{"level":"DEBUG","appenders":[{"name":"console","layout":"time=%d{yyyy-MM-dd}T%d{HH:mm:ss.SSSzzz} | lvl=%p | corr=%X{correlatorId} | trans=%X{transactionId} | srv=%X{service} | subsrv=%X{subservice} | function=%M | comp=Cygnus | msg=%C[%L] : %m%n"}]}}
$ curl -X GET "http://localhost:8081/admin/log?verbose=false"
{"level": "DEBUG"}

[CASES COMBINING PARAMETERS]
$ curl -X GET "http://localhost:8081/admin/log?verbose=false&transient=true"
{"level": "DEBUG"}

$ curl -X GET "http://localhost:8081/admin/log?verbose=false&transient=false"
{"level": "INFO"}

$ curl -X GET "http://localhost:8081/admin/log?verbose=true&transient=false"
{"success":"true","log4j":{"level":"INFO","appenders":[{"name":"LOGFILE","layout":"time=%d{yyyy-MM-dd}T%d{HH:mm:ss.SSSzzz} | lvl=%p | corr=%X{correlatorId} | trans=%X{transactionId} | srv=%X{service} | subsrv=%X{subservice} | function=%M | comp=Cygnus | msg=%C[%L] : %m%n"}]}}

$ curl -X GET "http://localhost:8081/admin/log?verbose=true&transient=true"
{"success":"true","log4j":{"level":"DEBUG","appenders":[{"name":"console","layout":"time=%d{yyyy-MM-dd}T%d{HH:mm:ss.SSSzzz} | lvl=%p | corr=%X{correlatorId} | trans=%X{transactionId} | srv=%X{service} | subsrv=%X{subservice} | function=%M | comp=Cygnus | msg=%C[%L] : %m%n"}]}}
```
* Assignee: @frbattid

### CODE REVIEW

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/nodes/CygnusApplication.java [hunk 1]
```diff
@@ -202,7 +202,8 @@ public static void main(String[] args) {
             File configurationFile = new File(commandLine.getOptionValue('f'));
             String agentName = commandLine.getOptionValue('n');
             boolean reload = !commandLine.hasOption("no-reload-conf");
-
+            
+            String configurationPath = configurationFile.getParent();
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/nodes/CygnusApplication.java]: White line after this one is required.

Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/nodes/CygnusApplication.java]: Fixed in  8ab92fd

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 2]
```diff
@@ -411,40 +414,104 @@ private void handleGetAdminLog(HttpServletRequest request, HttpServletResponse r
         try {
             Level level = LogManager.getRootLogger().getLevel();
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: This sentence should go within a section where transient==true.

Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Fixed in  018b9af

**Comment by frbattid** [comment]
LGTM

**Pull request merged**